pkg/apis/workflow/v1alpha1/generated.proto

```diff
@@ -503,6 +503,7 @@ message Parameter {
   optional string name = 1;
 
   // Default is the default value to use for an input parameter if a value was not supplied
+  // DEPRECATED - this field is never used
```

Comment:
```suggestion
  // DEPRECATED: This field is not used
```

Reply:
I actually wonder if we should just delete this field now?